Changes

```diff
@@ -97,6 +101,7 @@ Fixes
   - Fixed bug when deselecting a curve - the plug listing selection was not being updated
     to reflect the change.
 - PythonCommand : Prevented inadvertent modification of outer context (#3101).
+- Fixed widget signal/slot lifetime issues (#3179).
```

Comment:
Perhaps we should mention the new recommended way of managing widget connections?